spec/features/hub/create_organization_hierarchy_spec.rb

```diff
@@ -39,12 +39,14 @@
       expect(page).to have_selector("h1", text: "Oregano Org")
       expect(page).to have_text("Llama Library")
 
-      # Update a site
+      # Update the site
       click_on "Llama Library"
       fill_in "Name", with: "Lima Bean Library"
       click_on "Save"
+      expect(page).to have_selector("h1", text: "Lima Bean Library")
 
-      expect(page).to have_selector("h1", text: "Oregano Org")
+      # Navigate to the org
+      click_on "Oregano Org"
```

Comment:
I would expect the feature specs to test: 1) that the Unique Urls are not shown on the new page, 2) that a list of all the Unique Urls are shown on the edit page, 3) that you can edit an existing Unique Url, 4) that you can add a new Unique Url, 5) you can remove a Unique Url.

Reply:
This is a good idea. I'll punt on it for now.